src/Voron/Impl/Paging/Windows32BitsMemoryMapPager.cs

```diff
@@ -269,6 +269,16 @@ public override I4KbBatchWrites BatchWriter()
             return new Windows32Bit4KbBatchWrites(this);
         }
 
+        public override void DiscardPages(long pageNumber, int numberOfPages)
+        {
+            // REVIEW: Nothing to do here?.
```

Comment:
Probably nothing, we unmap it from memory entirely, so I would guess not.
However, for temp files, we might want to explicitly discard before unmap, to avoid writing to disk